jsconfig.eslint.json

```diff
@@ -0,0 +1,13 @@
+{
+	"compilerOptions": {
+		"noEmit": true,
+		"allowJs": true
+	},
+	"include": [
+		".eslintrc.js",
+		".prettierrc.js",
+		"babel.config.js",
+		"jest.config.js",
+		"webpack.config.js"
+	]
```

Comment:
This is required because of this issue
https://typescript-eslint.io/linting/troubleshooting/#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file